darwin-amd64/go.mod

```diff
@@ -1,11 +1,26 @@
-module github.com/duckdb/duckdb-go-bindings/darwin-amd64
```

Comment:
Revert this change - same for the other files.

Reply:
Ah yes, I have forgot. All platform binding module names had been fixed.